users-service/src/business/UsuarioBusiness.js

```diff
@@ -0,0 +1,45 @@
+const UsuarioRepository = require('../repository/UsuarioRepository');
+
+const RETORNO_SUCESSO = 1;
+
+module.exports = {
+   async validarUsuarioExistente(nome) {
+      const usuario = await this.findUserByName(nome);
+
+      if (usuario.length > 0) return true;
+
+      return false;
+   },
+   async addUser(nome, cargo, tecnologias) {
+      const usuario = await UsuarioRepository.addUser(nome, cargo, tecnologias);
+
+      return usuario;
```

Comment:
O que acha de retornar direto?

```suggestion
      return await UsuarioRepository.addUser(nome, cargo, tecnologias);
```